tlatools/org.lamport.tlatools/src/tla2sany/StandardModules/_TLCActionTrace.tla

```diff
@@ -0,0 +1,52 @@
+------- MODULE _TLCActionTrace ------
+LOCAL INSTANCE IOUtils
+LOCAL INSTANCE TLC
+LOCAL INSTANCE TLCExt
+LOCAL INSTANCE Sequences
+LOCAL INSTANCE SequencesExt
+LOCAL INSTANCE FiniteSets
+LOCAL INSTANCE FiniteSetsExt
+
+LOCAL _extends ==
+    LET M == FoldSet(LAMBDA a, acc: acc \cup {a[2].location.module}, 
+            {}, CounterExample.action)
+    IN FoldSet(LAMBDA module, acc: acc \o (IF acc = "" THEN "EXTENDS " ELSE ", ") \o module, 
+                "", M)
+
+LOCAL _next ==
+    LET Params(r) == FoldLeft(LAMBDA acc, p: acc \o (IF acc = "" THEN "" ELSE ", ") \o ToString(r.context[p]), 
+                              "", r.parameters)
+        Signature(r) == IF "parameters" \in DOMAIN r THEN r.name \o "(" \o Params(r) \o ")" ELSE r.name
+        ToDisjunct(i, r) == "\\/ " \o ToString(i) \o " = _idx /\\ " \o Signature(r) \o "\n"
+    IN TLCLiteral(FoldSet(LAMBDA a, acc: acc \o ToDisjunct(a[1][1], a[2]), 
+            "", CounterExample.action))
```

Comment:
I'm not entirely sure I understand the purpose of `TLCLiteral`. It looks like `ToString(TLCLiteral("x")) = "x"`, or more generally
```
\A s \in STRING: ToString(TLCLiteral(s)) = s
```
Is that accurate or did I misread something?

Reply:
Thanks for stopping me from doing something foolish.   The updated PR removes the commit that introduced `TLCLiteral`.  On a related note, it would be great to eventually have CommunityModules available as a test time dependency.